tox.ini

```diff
@@ -58,6 +59,7 @@ deps =
     # Twisted[http2] is required to import some files
     Twisted[http2]>=17.9.0
     pytest-flake8==1.1.1
+    # newer ones don't work: https://github.com/tholo/pytest-flake8/issues/87
```

Comment:
I wonder if it'd be simple enough to use flake8 directly, without pytest-flake8 plugin.

Reply:
I'll try, https://github.com/pypa/pep517/pull/147/files suggests it should be simple.